config/Zed/Oms/PayolutionSubProcesses/Capture.xml

```diff
@@ -19,7 +19,7 @@
         </transitions>
 
         <events>
-            <event name="capture payment" onEnter="true" manual="true" command="Payolution/Capture"/>
+            <event name="capture payment" manual="true" command="Payolution/Capture"/>
```

Comment:
Please make sure that there are no any objections from PO and customers side as it might have major affect on existing projects.